### DISCUSSION TITLE
v3.2.61

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#12056] [General]
## What's Changed
* Update vi translations by @datlechin in https://github.com/filamentphp/filament/pull/12014
* Update 02-listing-records.md by @andruu in https://github.com/filamentphp/filament/pull/12011
* Update korean translations by @corean in https://github.com/filamentphp/filament/pull/12013
* Conditionally apply margin to notification contents by @dododedodonl in https://github.com/filamentphp/filament/pull/12010
* Revert "Update 02-listing-records.md" by @danharrin in https://github.com/filamentphp/filament/pull/12025
* Add `handleRegistration()` method to register page by @dododedodonl in https://github.com/filamentphp/filament/pull/12020
* Fix docs on how to change pagination page when testing by @cheesegrits in https://github.com/filamentphp/filament/pull/12033
* fix: Notification action JS views by @danharrin in https://github.com/filamentphp/filament/pull/12041
* Fix issue #12045 : Unsaved changes alert is triggered when none ascii characters are used. by @abournavaud in https://github.com/filamentphp/filament/pull/12046
* Handle array states and values in assertTableColumnStateSet by @cheesegrits in https://github.com/filamentphp/filament/pull/12032
* fix: Group / summarizer date translations by @danharrin in https://github.com/filamentphp/filament/pull/12052
* Handle cases where Wizard Step is not found (hidden steps) by @AlexisSerneels in https://github.com/filamentphp/filament/pull/12054

**v3.2.60:** Work around Blade compilation bug with imports.

## New Contributors
* @corean made their first contribution in https://github.com/filamentphp/filament/pull/12013
* @abournavaud made their first contribution in https://github.com/filamentphp/filament/pull/12046

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.59...v3.2.61

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.61'>v3.2.61</a>.</em>

**Comment by raakkan** [comment]
installation error in fresh laravel 11 instllation with below error.

Problem 1
    - illuminate/console[v10.0.0, ..., v10.48.4] require nunomaduro/termwind ^1.13 -> found nunomaduro/termwind[v1.13.0, ..., v1.15.1] but these were not loaded, likely because it conflicts with another require.
    - filament/filament v3.2.0 requires illuminate/console ^10.0 -> satisfiable by illuminate/console[v10.0.0, ..., v10.48.4].
    - Root composer.json requires filament/filament 3.2 -> satisfiable by filament/filament[v3.2.0].

**Reply by danharrin** [reply]
It should be `^3.2` not `3.2` for the version constraint

**Reply by raakkan** [reply]
im using this command to install filament from the documentation  "composer require filament/filament:"^3.2" -W"

**Reply by danharrin** [reply]
Try just `composer require filament/filament`

**Reply by raakkan** [reply]
working thanks for your reply
